logger/logger.md

```diff
@@ -1,20 +1,38 @@
 # GarageLogger
+Proposal by Peter Lillian
 
-The current logger is simply a collection of global variables that needs to be replaced with a new class. I'm calling this new class **GarageLogger**.
+## Background
 
-![GarageLogger](GarageLogger.svg)
+The current logger is simply a collection of global variables that needs to be replaced with a new object-oriented API that is easier to work with. I'm calling this new class **GarageLogger**.
```

Comment:
This is a nit, but there's no point in putting the prefix "Garage" on any class in garage. So this would be called logger.